### DISCUSSION TITLE
how to ignore a particular route for layouts in sveltekit?

### DISCUSSION THREAD

**Discussion by tryggs** [sveltejs/kit#5754] [Q&A] [answered]
Hello, I am using layout in sveltekit but i do not want to use it for one route.

```
//__layout.svelte

<script>
  import "../app.css";
</script>

<slot></slot>
```

and I have a route called `[slug].svelte` where I don't want layouts to apply.
How would this be done?

thanks in advance!

**Comment by teemingc** [comment, marked as answer]
Hi tryggs,

You will have to create a "reset" layout and append the name of that layout to your route. For example:
```
//__layout-reset.svelte

<slot />
```

Then, modify your route name to `[slug]@reset.svelte` to apply the "reset" layout to it.
This should prevent it from inheriting from `__layout.svelte`.

[Documentation for layout inheritance](https://kit.svelte.dev/docs/layouts#named-layouts-inheritance-chains)

Hope that helps :)

**Reply by tryggs** [reply]
Thank you!
